packages/terra-menu/src/clinical-lowlight-theme/Menu.module.scss

```diff
@@ -0,0 +1,78 @@
+:local {
+  .clinical-lowlight-theme {
```

Comment:
Let's break the MenuItem theme vars into their own file.

Reply:
Done cf26d44